truco-front/src/hooks/usePusherListeners.ts

```diff
@@ -153,10 +155,12 @@ export const usePusherListeners = (
         })
 
         gameManager.events.addOnFriendRequestListener((friendUser: User) => {
+            toast(friendUser.username + " sent you a friend request!")

```

Comment:
after this, you can find the "onFriendRequestAcceptedListener" that triggers when someone accepts a friend request, maybe you can use toast there, too.

Reply:
Added